src/blocks/pacman.rs

```diff
@@ -304,20 +312,26 @@ pub async fn run(config: Config, mut api: CommonApi) -> Result<()> {
 #[derive(Debug, PartialEq, Eq)]
 enum Watched {
     None,
-    Pacman,
+    Pacman(Option<String>),
     Aur(String),
-    Both(String),
+    Both(String, Option<String>),
 }
 
-async fn check_fakeroot_command_exists() -> Result<()> {
-    if !has_command("fakeroot").await? {
-        Err(Error::new("fakeroot not found"))
-    } else {
-        Ok(())
-    }
+async fn get_pacman_available_updates(pacman_command: &Option<String>) -> Result<String> {
+    let Some(pacman_command) = pacman_command else {
+        return get_pacman_available_updates_fakeroot().await
+    };
+    let stdout = Command::new("sh")
```

Comment:
If a pacman_command is supplied then there's no updating the `sync` dir (neither `/var/lib/pacman/sync` nor `/tmp/checkup-db-i3statusrs/sync`).